src/components/date-range-picker/date-range-picker.spec.ts

```diff
@@ -102,10 +103,49 @@ describe('Date Range Picker', () => {
         expect(getDate(1)).toBeNull();
     });
 
+    it('should allow a half time zone to be set if it is present in the timezone array', async() => {
+        component.showTime = true;
+        component.showTimezone = true;
+        component.timezones = [{ name: 'GMT-3.5', offset: 210 }, { name: 'GMT-3', offset: 180 }, { name: 'GMT-2', offset: 120 }, { name: 'GMT-1', offset: 60 }, { name: 'GMT', offset: 0 }];
+        component.startTimezone = { name: 'GMT-3.5', offset: 210 };
+        component.endTimezone = { name: 'GMT-3.5', offset: 210 };
+
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        await expect(getGMTValue(0).getAttribute('ng-reflect-value')).toBe('GMT-3.5');
+        await expect(getGMTValue(1).getAttribute('ng-reflect-value')).toBe('GMT-3.5');
+    });
+
+    it('should default to local timezone', async() => {
+        component.showTime = true;
+        component.showTimezone = true;
+
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        await expect(getGMTValue(0).getAttribute('ng-reflect-value')).toBe('GMT');
```

Comment:
I don't think we should be relying on reflect values. These are internal Angular debug properties and while this may work fine now these may change and in Angular 9 (with the CLI) tests are run in AOT mode, which may strip out these debug values